source/MaterialXTest/ShaderGen.cpp

```diff
@@ -1079,10 +1079,10 @@ TEST_CASE("Hello World", "[shadergen]")
         // Write out to file for inspection
         // TODO: Use validation in MaterialXRender library
         std::ofstream file;
-        file.open(RESULT_DIRECTORY + shader->getName() + "_graph.vert");
+        file.open(RESULT_DIRECTORY + shader->getName() + "_graph_vs.glsl");
```

Comment:
Changed naming convention for output files to "vs.glsl" for vertex shaders and "ps.glsl" for pixel shaders. It's then more clear that this is GLSL code. It's the convention used in MaterialXView when saving shaders to file.